.docker/app/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nginx:1.17.0-alpine
+
+RUN rm /etc/nginx/conf.d/default.conf
+
+COPY .docker/app/nginx.conf /etc/nginx/nginx.conf
+
+WORKDIR /app
+
+COPY . /app
```

Comment:
you just have to copy the `public` dir, no ?
`COPY ./public /app/public`

Reply:
Mmh nope I also need to copy the nginx conf, and I better like to do it with a .dockerignore file, like you explained before. It's cleaner IMHO

Reply:
As the nginx conf is copied in the previous instructions, I think  you can change this line for
`COPY ./public /app/public` otherwise you'll also copy the `./.docker/app/nginx.conf` into `/app/public/.docker/app/nginx.conf` :/

Better explicitly tell what you want to copy when building prod images (this will avoid this kind of errors).

The `.dockerignore` file is mainly used to indicate what should be sent from the current dir to the docker context when building the image. The less you send, the better it is.